xray/config_test.go

```diff
@@ -110,6 +110,15 @@ func TestEnvironmentConfigureParameters(t *testing.T) {
 
 }
 
+func TestInvalidEnvironmentDaemonAddress(t *testing.T) {
+
```

Comment:
nitpick.... could you remove the empty line at the beginning and end of the method?